cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/hbase/AbstractHBaseDataSetAdmin.java

```diff
@@ -234,7 +234,7 @@ protected void addCoprocessor(HTableDescriptorBuilder tableDescriptor, Class<? e
    *
    * @return true if the table descriptor is modified, that is, whether update is needed.
    */
-  protected abstract boolean needsUpdate(HTableDescriptor tableDescriptor);
+  protected abstract boolean needsUpdate(HTableDescriptor tableDescriptor, HTableDescriptorBuilder descriptorBuilder);
```

Comment:
Add `@param` to tell what are `tableDescriptor` and `descriptorBuilder` are for.